barretenberg/cpp/src/barretenberg/circuit_checker/ultra_circuit_builder.test.cpp

```diff
@@ -1,4 +1,5 @@
-#include "ultra_circuit_builder.hpp"
+#include "barretenberg/proof_system/circuit_builder/ultra_circuit_builder.hpp"
```

Comment:
It feels very weird that the tests for ultra are not in the same directory as the builder. Is it because you didn't want to create a circular dependency?